Makefile

```
@@ -97,7 +113,17 @@ test-e2e-matrix: ## Run e2e tests against both shard configs (single-shard, mult
 	$(MAKE) clean-e2e
 	E2E_SHARD_CONFIG=multi-shard  $(MAKE) test-e2e
 
-.PHONY: e2e-cleanup
+.PHONY: test-e2e-kcp-matrix
+test-e2e-kcp-matrix: ## Run e2e against every kcp version in KCP_VERSIONS.
```

checkmake (0.3.2)

[warning] Target body for "test-e2e-kcp-matrix" exceeds allowed length of 5 lines (7).

(maxbodylength)